data/json/recipes/ammo/weldgas.json

```diff
@@ -208,6 +208,6 @@
       [ [ "forge", 200 ] ]
     ],
     "proficiencies": [ { "proficiency": "prof_metalworking" } ],
-    "components": [ [ [ "material_aluminium_ingot", 1 ] ] ]
+    "components": [ [ [ "material_aluminium_ingot", 1 ], [ "scrap_aluminum", 15 ] ] ]
```

Comment:
both amount ot the same mass?

Reply:
Actually, my bad, they do not. It should be 13 not 15, I'll change this right away